test/wt_hang_analyzer/wt_hang_analyzer.py

```diff
@@ -626,5 +632,21 @@ def main():
     if trapped_exceptions:
         sys.exit(1)
 
+# Remove the lowest bit from the core dump mask. The lowest bit is for
+# anonymous private mappings (see: `man core`). These mappings are used by
```

Comment:
The backticks are a bit unusual in a comment and probably not required here.

Reply:
fixed in 49ee77f